src/vorta/application.py

```diff
@@ -23,7 +23,12 @@
 
 logger = logging.getLogger(__name__)
 
-APP_ID = TEMP_DIR / "socket"
+# If config.TEMP_DIR is None, this means that the app got launched withoug going through main
```

Comment:
Typo

Reply:
Sorry, what does this mean? I can't find the typo.

Reply:
It sais `withoug` instead of `without`.